docs/content/guide/migration.ngdoc

```diff
@@ -1683,6 +1683,79 @@ ngModelCtrl.$formatters.push(function(value) {
 });
 ```
 
+#### ngPattern and pattern
+
+Due to [0e001084](https://github.com/angular/angular.js/commit/0e001084ffff8674efad289d37cb16cc4e46b50a),
+The `ngPattern` and `pattern` directives will validate the regex
+against the `viewValue` of `ngModel`, i.e. the value of the model
```

Comment:
You should prefix viewValue, modelValue with `$` as this is how they are stored in the model